File: Data/Entities/file.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Data.Entities
{
    public class file
    {
        public int id { get; set; }
        public int userId { get; set; }
        public string fileName { get; set; }
        public string fileType { get; set; }
        public Byte[] data { get; set; }

        /// <summary>
        /// Có được sử dụng ?
        /// </summary>
        public bool isUsed { get; set; }
        public virtual user Users { get; set; }
    }
}

File: demo_user/ClientApp/src/app/Models/FileModel/file.ts
export class file{
  public id:number;
  public userId:number;
  public fileName:string;
  public fileType:string;
  public data:string;
  public isUsed:boolean;

  constructor(){
    this.id = 0;
    this.userId = 1;
    this.fileName = "";
    this.fileType = "";
    this.data = "";
    this.isUsed = true;
  }
}
